Source: Timo Gurr <tgurr@exherbo.org>
Upstream: Wait until the CMake conversion settled a bit
Reason: Don't try to install nonexistent files

diff -Naur doxygen-1.8.14/doc/CMakeLists.txt doxygen-1.8.14.new/doc/CMakeLists.txt
--- doxygen-1.8.14/doc/CMakeLists.txt	2017-10-31 20:36:00.000000000 +0100
+++ doxygen-1.8.14.new/doc/CMakeLists.txt	2018-07-20 13:35:30.981584018 +0200
@@ -170,13 +170,3 @@
         "${PROJECT_BINARY_DIR}/man/doxyindexer.1"
         DESTINATION ${CMAKE_INSTALL_MANDIR}/man1
 )
-
-install(FILES
-        "${PROJECT_BINARY_DIR}/latex/doxygen_manual.pdf"
-        DESTINATION "${CMAKE_INSTALL_PREFIX}/${DOC_INSTALL_DIR}"
-)
-
-install(DIRECTORY
-        "${PROJECT_BINARY_DIR}/html"
-        DESTINATION "${CMAKE_INSTALL_PREFIX}/${DOC_INSTALL_DIR}"
-)
